ecosystem/platform/server/app/views/it2s/show.html.erb

```diff
@@ -17,59 +17,69 @@
       </h2>
     <% end %>
 
-    <div class="flex flex-wrap flex-col xl:flex-row gap-8" data-controller="shake">
+    <div class="flex flex-wrap flex-col gap-8 max-w-prose" data-controller="shake">
       <% @steps.each_with_index do |step, i| %>
         <div data-action="<%= step.disabled ? 'click->shake#shake' : '' %>" data-target="<%= !step.disabled && !step.completed ? 'shake.content' : '' %>" class="bg-neutral-800 rounded-md flex flex-col flex-1 <%= step.disabled ? 'cursor-not-allowed select-none opacity-25' : '' %>">
-          <h3 class="font-mono uppercase mb-2 bg-neutral-700 text-teal-400 px-4 py-2 rounded-t-md text-sm">Step <%= i + 1 %></h3>
-          <div class="px-8 py-4 flex flex-col flex-1">
-            <h4 class="text-teal-400 font-mono uppercase text-2xl mb-4">
+          <h3 class="font-mono uppercase mb-2 bg-neutral-700 text-teal-400 px-8 lg:px-12 py-2 rounded-t-md text-sm">Step <%= i + 1 %></h3>
+          <div class="px-8 lg:px-12 pt-4 pb-8 flex flex-col lg:flex-row lg:gap-16 items-center flex-1">
+            <div class="w-full flex-1">
+              <h4 class="text-white font-mono uppercase text-2xl mb-4">
+                <% if step.name == :connect_discord %>
+                  Connect Discord
+                <% elsif step.name == :survey %>
+                  Complete Survey
+                <% elsif step.name == :node_registration %>
+                  Node Registration
+                <% elsif step.name == :identity_verification %>
+                  Identity Verification
+                <% end %>
+              </h4>
+
+              <div class="mb-4"><%= render DividerComponent.new %></div>
+
+              <p class="mb-8 lg:mb-0 font-light">
               <% if step.name == :connect_discord %>
-                Connect Discord
+                Sign in to your Discord to continue node verification.
+              <% elsif step.name == :survey %>
+                Lorem ipsum dolor sit amet
```

Comment:
"Please tell us a bit more about yourself, and your experience with Aptos"

Reply:
Done.